### PR TITLE
fix(double-requests): removed double friends requests

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Prevent double friend requests to be shown (we keep the first one)

**Which issue(s) this PR fixes** 🔨
AP-1708

**Special notes for reviewers** 🗒️

**Additional comments** 🎤
Although the user shouldn't be able to make multiple requests to another user, sometimes it seem to happen (i wasn't able to replicate it tho) and this PR aims to remove all multiple requests, keeping just the first made.
Also added the style for the "success" button, because was missing (the green button that appears to confirm the friend request).

### CODE REVIEW

**Comment by stavares843** [comment]
if we keep the first one instead of the latest one, is there a possibility that one is somewhat expired?

**Comment by ThomBos** [comment]
> if we keep the first one instead of the latest one, is there a possibility that one is somewhat expired?

no no, friends requests don't expire, i had a talk with Manuel for that 👍

**Review by Jekrimo — APPROVED** [review]

File: store/friends/actions.ts [hunk 1]
```diff
@@ -75,8 +76,8 @@ export default {
       }),
     )
 
-    commit('setIncomingRequests', incomingRequests)
-    commit('setOutgoingRequests', outgoingRequests)
+    commit('setIncomingRequests', friendListFilterDuplicates(incomingRequests))
+    commit('setOutgoingRequests', friendListFilterDuplicates(outgoingRequests))
   },
```
Comment (molimauro) [inline store/friends/actions.ts:81]: simply use `uniqBy(list, 'requestId')` directly

**Review by molimauro — DISMISSED** [review]
minor

**Comment by molimauro** [comment]
@ThomBos just to understand, is this a final fix or is this a workaround? :)

**Comment by ThomBos** [comment]
> @ThomBos just to understand, is this a final fix or is this a workaround? :)

it's a workaround. i wasn't able to replicate the bug yet 🧠💥

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Got added by 20 users without a single duplicate 🔥

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
🎉

**Pull request merged**